### PR TITLE
New arc start screen

### PR DESCRIPTION
**Pull request by ManavWednesday**

### CODE REVIEW

File: presentation-entity/src/main/java/com/wednesday/template/presentation/weather/FavoriteCity.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+data class FavoriteCity(
+    val cityId: Int,
+    val title: String,
+    val displayTitle: UIText,
+    val locationType: String,
+    val displayLocationType: UIText,
+    val latitude: String,
+    val isFavourite: Boolean,
+    val current_temp:String
+): UIListItemBase(id = "UICity $cityId")
```
Comment (shounak-mulay) [inline presentation-entity/src/main/java/com/wednesday/template/presentation/weather/FavoriteCity.kt:17]: Prefix presentation classes with UI
```suggestion
@Parcelize
data class UIFavoriteCity(
    val cityId: Int,
    val title: String,
    val displayTitle: UIText,
    val locationType: String,
    val displayLocationType: UIText,
    val latitude: String,
    val isFavourite: Boolean,
    val current_temp:String
): UIListItemBase(id = "UICity $cityId")
```

File: presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartFragment.kt [hunk 2]
```diff
@@ -1,26 +1,40 @@
...
+    override val bindingProvider: BindingProvider<FragmentStartBinding> = FragmentStartBinding::inflate
 
     override fun onEffect(effect: Effect) {
-        TODO("Not yet implemented")
+    }
+    
+    override fun onViewCreated(binding: FragmentStartBinding) {
+        super.onViewCreated(binding)
+        Log.d("*************","Calledddddddddddddddd")
+        initializeList(binding)
```
Comment (shounak-mulay) [inline presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartFragment.kt:29]: Let's remove this. We will use the `ListComponent` and update it in `onState`

File: presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartScreenViewModel.kt [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
+) : BaseViewModel<StartFragmentScreen, StartScreenState>(),
     IntentHandler<StartScreenIntent> {
-
+    
     override fun getDefaultScreenState(): StartScreenState {
-        TODO()
+        return StartScreenState(
+            toolbar = UIToolbar(
+                title = UIText { block("Search") },
+                hasBackButton = true,
```
Comment (shounak-mulay) [inline presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartScreenViewModel.kt:20]: This is the start page. This should not have a back button.

File: presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartFragment.kt (hunk 2 above)
Comment (ManavWednesday) [reply presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartFragment.kt:29]: Yeah I have removed it.

File: presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartScreenViewModel.kt (hunk 3 above)
Comment (ManavWednesday) [reply presentation/src/main/java/com/wednesday/template/presentation/weather/start/StartScreenViewModel.kt:20]: Done

File: presentation-entity/src/main/java/com/wednesday/template/presentation/weather/FavoriteCity.kt (hunk 1 above)
Comment (ManavWednesday) [reply presentation-entity/src/main/java/com/wednesday/template/presentation/weather/FavoriteCity.kt:17]: Done

**Pull request merged**